config/GermanDBPediaSearch.properties

```diff
@@ -17,12 +17,15 @@ kb.endpoint=http://de.dbpedia.org/sparql
 kb.ontologyURI=file:///d:/Documents/Odalic/Sti/resources/dbpedia_ontology_2014.owl
 # lists of predicates for the knowledge base
 # Multiple configuration files can be separated by the "|"
-kb.structure=config\\enums\\de.dbpedia.properties|config\\enums\\dc.properties|config\\enums\\foaf.properties|config\\enums\\rdf.properties|config\\enums\\owl.properties
+kb.structure=config\\enums\\de.dbpedia.properties|config\\enums\\dc.properties|config\\enums\\foaf.properties|config\\enums\\de.dbpedia.rdf.properties|config\\enums\\de.dbpedia.owl.properties
 # cache template for the knowledge base
 kb.cacheTemplatePath=d:\\Documents\\Odalic\\Cache\\empty\\
 # The language suffix used for string matching
 kb.languageSuffix=@de
 
 # Vistuoso specific settings
 # Vistuoso fulltext search
-kb.useBifContains=true
\ No newline at end of file
+kb.useBifContains=true
+
+# Specifies, that in fulltext search, the url itself will also be considered as label (useful in knowledge bases, that have no labels for some groups of resources).
+kb.treatUrlAsLabel=true
```

Comment:
This is a remnant of some experiments I tried with the full-text search. I will remove this change.